reana_client/validation.py

```diff
@@ -343,6 +344,28 @@ def _warn_not_defined_parameters(
         )
 
 
+def _validate_cwl_parameters(reana_yaml):
+    """Validate input parameters for CWL workflows.
+
+    :param reana_yaml: REANA YAML specification.
+    """
+
+    # Check if cwltool is installed.
+    run_command("cwltool --version", display=False, return_output=True)
+
+    cwl_directory = reana_yaml["inputs"]["directories"][0]
```

Comment:
Moderate: We can't just assume that there is always a `directories`, nor that the first item it's gonna be always `workflow/cwl`. `directories` is just an optional input property to upload all that's inside to the cluster. Instead, we should better look at [`workflow.file`](https://docs.reana.io/reference/reana-yaml/#reanayaml-workflow) and from there maybe look at possible references.

Check for instance `reana-demo-cdci-crab-pulsar-integral-verification` that doesn't have `directories`.